File: ClientePy/entity/Cliente.py
class Cliente:
    def __init__(self, id_cliente, nome, telefone):
        self.__id_cliente = id_cliente
        self.__nome = nome
        self.__telefone = telefone


    def getId_cliente(self):
        return self.__id_cliente

    def setId_cliente(self, id_cliente):
        self.__id_cliente = id_cliente

    def getNome(self):
        return self.__nome

    def setNome(self, nome):
        self.__nome = nome

    def getTelefone(self):
        return self.__telefone

    def setTelefone(self, telefone):
        self.__telefone = telefone

    def getClass(self):
        cliente = Cliente()
        return cliente.__dict__.update()

File: Servidor/src/br/entity/Cliente.java
package entity;

public class Cliente {
    private int idCliente;
    private String nome;
    private String telefone;

    public Cliente(int idCliente, String nome, String telefone) {
        this.idCliente = idCliente;
        this.nome = nome;
        this.telefone = telefone;
    }

    public String getNome() {
        return nome;
    }

    public void setNome(String nome) {
        this.nome = nome;
    }

    public String getTelefone() {
        return telefone;
    }

    public void setTelefone(String telefone) {
        this.telefone = telefone;
    }

    public int getIdCliente() {
        return idCliente;
    }
}